prime-router/src/main/kotlin/Translator.kt

```diff
@@ -133,16 +133,20 @@ class Translator(private val metadata: Metadata, private val settings: SettingsP
         )
         if (routingFilteredReport.isEmpty()) return routingFilteredReport
 
-        // Do conditionFiltering on the routingFilteredReport
-        val conditionFilteredReport = filterByOneFilterType(
-            routingFilteredReport,
-            receiver,
-            organization,
-            ReportStreamFilterType.CONDITION_FILTER,
-            trackingElement,
-            doLogging = true
-        )
-        if (conditionFilteredReport.isEmpty()) return conditionFilteredReport
+        var conditionFilteredReport = routingFilteredReport
+        // condition filter can only be used on UP
+        if (receiver.topic == Topic.FULL_ELR || receiver.topic == Topic.FULL_ELR_TEST) {
+            // Do conditionFiltering on the routingFilteredReport
+            conditionFilteredReport = filterByOneFilterType(
+                routingFilteredReport,
+                receiver,
+                organization,
+                ReportStreamFilterType.CONDITION_FILTER,
```

Comment:
Ran TranslatorTests with coverage and this change doesn't seem to be covered on the unit tests.

Reply:
How do I run the tests with coverage so that I can see what you are seeing?

Reply:
Alrighty, more tests have been added that should cover this.